src/styles/blocks/container.scss

```
@@ -0,0 +1,56 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 30px 5px;
+  width: getWidth($box-size, $number-boxes, $gap, $padding);
+  box-sizing: border-box;
+
+  $days:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  &__box {
+    @include box($box-size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $background-color;
+    border: 1px solid black;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $font-size;
+    transition: $background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color;
+      transform: translateY(-20px);
+      transition: 0.5s;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+  }
+
+  @each $name, $index in $days {
+    &--start-day_#{$name} > :first-child {
+      margin-left: $box-size * $index + $gap * ($index);
+    }
+  }
+  @for $last-day from 28 through 31 {
+    &--month-length_#{$last-day} {
+      .container__box:nth-child(n + #{$last-day + 1}) {
```

it should works as well
```suggestion
    &--month-length_#{$last-day}  &__box:nth-child(n + #{$last-day + 1}) {
```

---

I had to redo because nothing was working properly